### ISSUE TITLE
Couldn't found "(dotenv)" while running "node app.js"

### ISSUE THREAD

**Issue by quantoh7162** [fikiismyname/tikdo#3]
Hi Fikiismyname,

I had tried to run the test on "app.js" with command line "node app.js", it shows couldn't find the ("_[dotenv]()_") file, thanks for advises.

**Comment by fikiismyname** [comment]
> Hi Fikiismyname,
> 
> I had tried to run the test on "app.js" with command line "node app.js", it shows couldn't find the ("_dotenv_") file, thanks for advises.

have you followed my readme? by renaming the .env.example file to .env and filling in your telegram token bot, then run `npm start`

**Comment by quantoh7162** [comment]
Yes, I do change the “.env.example” to “.env”, but when i run npm start, it
shows this, any solutions, thank you?

**Comment by quantoh7162** [comment]
[](https://github.com/dappuniversity/price-bot/issues/20)

**Comment by quantoh7162** [comment]
finally i'm able to run the Telegram Bot, the command is working, but when i input both China version Tiktok (DouYin) and Native version of Tiktok URLs, both channels are not working, the bot command "/video tiktokURL" is not working, please revised or updates, thank you very much~ 👍 💯

**Comment by fikiismyname** [comment]
> finally i'm able to run the Telegram Bot, the command is working, but when i input both China version Tiktok (DouYin) and Native version of Tiktok URLs, both channels are not working, the bot command "/video tiktokURL" is not working, please revised or updates, thank you very much~ 👍 💯

Hi, thanks for reminding me and I fixed this problem [3b9acb2](https://github.com/fikiismyname/tikdo/commit/3b9acb2cbf370b935bc14042f8ca6f13de7db1be), the reason was I forgot to set env prefix for music and video
try to fill in your .env file to be like this

```
BOT_TOKEN="Telegram BOT Token"
API_URL="https://api.douyin.wtf/api"

PREFIX_VIDEO=video
PREFIX_MUSIC=music
```

I hope your problem can be solved
